docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+   app:
+       build:
+           context: .
+       ports:
+           - "8001:8000"
+       volumes:
+           - ./:/app
+           - my_media:/files/media
+       command: >
+           sh -c "python manage.py wait_for_db &&
+                  python manage.py migrate &&
+                  python manage.py runserver 0.0.0.0:8000"
+       env_file:
+           - .env
+       depends_on:
+           - db
+
+   db:
+       image: postgres:14-alpine
+       restart: always
+       ports:
+           - "5433:5432"
+       env_file:
+           - .env
+       volumes:
+           - my_db:$PGDATA
```

Comment:
The `volumes` configuration for the `db` service uses `$PGDATA`. Ensure that the `PGDATA` environment variable is correctly set in the `.env` file or elsewhere in your configuration.